src/api/app/views/webui/notifications/index.html.erb

```diff
@@ -8,7 +8,7 @@
   <p><%= @pagetitle %></p>
 
   <%= form_tag(bulk_update_notifications_path, { method: :patch, id: 'notification_form'}) do %>
-      <%= render partial: 'roles' %>
+      <%= render partial: 'roles', locals: { user: nil }  %>
```

Comment:
I think that here is the same case as my other comment in the view src/api/app/views/webui/user/notifications.html.erb. instead off passing nil hardcoded it should come from the controller action as a @user variable